spec/feature/registeration_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'capybara/rspec'
+
+feature "Chitter user behaviour" do
+
+  scenario "Users can sign up for Chitter" do
+    visit '/'
+    click_link "Sign Up"
+
+    expect { sign_up }.to change(User, :count).by(1)
```

Comment:
Be consistent in your expectations.  In `post_peep_spec` you tested that `peep.count` should be 1.